### PR TITLE
Allow adding external link to banner page

### PR DESCRIPTION
**Pull request by Ntu2koTrevor**

### CODE REVIEW

**Comment by Ntu2koTrevor** [comment]
@miltontony Please review

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/core/tags/bannerpages.html [hunk 1]
```diff
@@ -5,12 +5,14 @@
...
+	    {% if bannerpage.banner_link_page %}
+	    <a href="{% pageurl bannerpage.banner_link_page %}">{% image bannerpage.banner width-320 %}</a>
+	    {% else %}
+	    {% image bannerpage.banner width-320 %}
+	    {% endif %}
     {% else %}
-    {% image bannerpage.banner width-320 %}
-    {% endif %}
-  {% endif %}
+    {% external_link %}
```
Comment (saeedmb8) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/core/tags/bannerpages.html]: @Ntu2koTrevor this should be `{% bannerpage.external_link %}` also please implement it in the same way `bannerpage.banner_link_page` has been implemented. Above line 10 you need to add another else statement checking if the banner has an external_link and then adding the page URL for the external link

**Comment by saeedmb8** [comment]
@Ntu2koTrevor I've made a small comment, please also add test for the external link that you added

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/core/tags/bannerpages.html [hunk 2]
```diff
@@ -5,12 +5,14 @@
...
-    {% if bannerpage.banner_link_page %}
-    <a href="{% pageurl bannerpage.banner_link_page %}">{% image bannerpage.banner width-320 %}</a>
-    {% else %}
-    {% image bannerpage.banner width-320 %}
-    {% endif %}
-  {% endif %}
+	    {% if bannerpage.banner_link_page %}
+	    <a href="{% pageurl bannerpage.banner_link_page %}">{% image bannerpage.banner width-320 %}</a>
+	    {% elif bannerpage.external_link %}
+	    <a href="{% pageurl bannerpage.external_link%}">{% image bannerpage.banner width-320 %}</a>
```
Comment (miltontony) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/core/tags/bannerpages.html]: add the following: `target="_blank"`

**Comment by miltontony** [comment]
👍

**Comment by saeedmb8** [comment]
@Ntu2koTrevor Please create a new PR and add tests for this PR! We have missed to add the migration for the field that you added, we could have picked this up if we had tests. you also need to test the functionality that you add locally to always make sure it works as you expect it :) cc. @miltontony

**Pull request merged**